lib/tasks/add_blocklisted_user.rake

```diff
@@ -0,0 +1,14 @@
+desc 'Adds a user to the blocklist. ' \
```

Comment:
Is there any reason why the term `blacklisted` is used in the User model and `blocklist/blocklisted` is used here instead? I think it'd be nice to use the same term for both

Reply:
Also, I agree with what @adlamas said, a spec for this task would be nice

Reply:
agree. and prefer `blocklist` over `blacklist` (and `allowlist` over `whitelist`) rubydoc.info/gems/rubocop/RuboCop/Cop/Naming/InclusiveLanguage

Reply:
Old code use the deprecated terms, like in the model attribute. 
For this changes I use the new terminology.
Haven't migrated existing usage. Maybe we should create some issues to rename things?